apache/zeppelin
# [gh-pages][ZEPPELIN-2055] Add "What's new in 0.7" and Twitter feed to website main page

### What is this PR for?
There are lots of improvement in upcoming release 0.7(as mentioned in #1972 (release note)). To emphasize them to users and make contents enrich Zeppelin official website, I added some description \w gif images. And also added Twitter news feed at the bottom of the main page. 

### What type of PR is it?
gh-pages | Improvement

### Todos
* [x] - need to take care of gif images' alignment when screen width is small 

### What is the Jira issue?
[ZEPPELIN-2055](https://issues.apache.org/jira/browse/ZEPPELIN-2055)

### How should this be tested?
Build `gh-pages` branch as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#build-website). Or see the below gif sreenshot img :) 

### Screenshots (if appropriate)
 - Before 
![before](https://cloud.githubusercontent.com/assets/10060731/22618943/ee52b5ea-eb2c-11e6-896a-f90a50f41998.gif)

 - After 
![after](https://cloud.githubusercontent.com/assets/10060731/22623919/66aeaa22-ebae-11e6-8835-d5680fe61697.gif)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
